test/cli/fixtures/expected/tap-outputs.js

```diff
@@ -161,8 +162,7 @@ not ok 1 global failure
     at done (.*)
     at advanceTestQueue (.*)
     at Object.advance (.*)
-    at unblockAndAdvanceQueue (.*)
-    at <anonymous>
+    at unblockAndAdvanceQueue (.*)(\n    at <anonymous>)*
```

Comment:
Should this end with a `?` (if only zero or one of those show up)? Or could we see more than one?

Reply:
yep, this should.